app/controllers/contacts_controller.rb

```diff
@@ -0,0 +1,6 @@
+# Provide API for retrieving Contacts data
+class ContactsController < ApplicationController
+  def today
+    @contacts = ContactsService.new.contacts_for_team('Helpdesk', "#{Time.zone.today} 00:00:00", "#{Time.zone.today} 23:59:59")
```

Comment:
Ruby/Rails seems to offer `beginning_of_day` ja `end_of_day` that could show intent better.